WebJobs.Script.proj

```diff
@@ -6,6 +6,11 @@
     <SkipStrongNamesExe>packages\Microsoft.Web.SkipStrongNames.1.0.0\tools\SkipStrongNames.exe</SkipStrongNamesExe>
     <SkipStrongNamesXml>tools\SkipStrongNames.xml</SkipStrongNamesXml>
     <PublishPath Condition=" '$(PublishPath)' == '' ">bin</PublishPath>
+    <BuildNumber Condition=" '$(BuildNumber)' == '' ">0</BuildNumber>
+    <SiteExtensionVersion>1.0.$(BuildNumber)</SiteExtensionVersion>
```

Comment:
I'm fine with it jumping around. That what we do with Kudu. So only needed change I think is to use `0.1.{build}` instead of `1.0.{build}`